### DISCUSSION TITLE
How can we get Users in a Group in form flow? Is it possible to get users along with their attributes defined in keycloak?

### DISCUSSION THREAD

**Discussion by sreehari-aot** [AOT-Technologies/forms-flow-ai#1176] [Q&A] [answered]
Is there any option to fetch users that belong to a particular group in formsflow.ai

**Comment by sreehari-aot** [comment, marked as answer]
Please refer to our API documentation
![image](https://user-images.githubusercontent.com/93634377/214230846-0c2977cf-2e9c-45e3-b546-0c8d004e70c3.png)
![image](https://user-images.githubusercontent.com/93634377/214230873-190b5c33-3aee-46d9-bd2f-1b2bcbde859b.png)

**Reply by abhilash-aot** [reply]
We will be releasing this as a new feature in our next release. Giving designers/Super Admin roled users permission to create groups and associate users to group/role from the formsflow.ai UI itself and no need to go into keycloak UI for that.
